### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by tsingcbx99**

### CODE REVIEW

File: examples-da/unsupervised/self_ensemble.py [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+    tgt_accs = AverageMeter('Tgt Acc', ':3.1f')
+
+    progress = ProgressMeter(
+        args.iters_per_epoch,
+        [batch_time, data_time, cls_losses, cons_losses, cls_accs, tgt_accs],
+        prefix="Epoch: [{}]".format(epoch))
+
+    # switch to train mode
+    model.train()
+    teacher.train()
```
Comment (JunguangJiang) [inline examples-da/unsupervised/self_ensemble.py:239]: should teacher be in train mode?
